sqlite3_storage_plugin/src/rosbag2/storage/sqlite/sqlite_storage.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright (c) 2018,  Bosch Software Innovations GmbH.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include "sqlite_storage.hpp"
+
+#include <iostream>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace rosbag2
+{
+
+SqliteStorage::SqliteStorage(const std::string & database_name, bool shouldInitialize)
```

Comment:
according to the ROS 2 style guide, this variable should be called `should_initialize`